packages/clerk-js/src/core/resources/DisplayConfig.ts

```diff
@@ -53,7 +53,6 @@ export class DisplayConfig extends BaseResource implements DisplayConfigResource
     this.userProfileUrl = data.user_profile_url;
     this.afterSignInUrl = data.after_sign_in_url;
     this.afterSignUpUrl = data.after_sign_up_url;
-    this.afterSignOutUrl = data.after_sign_out_url;
```

Comment:
❓ can we exclude the changes of `afterSignOutUrl / after_sign_out_url ` from this commit / PR ?